AAI/AAIConnectProfile.md

```diff
@@ -511,128 +503,169 @@ JWTs follow <https://tools.ietf.org/html/rfc7515> (JWS).
 ```
 {
  "typ": "JWT",
  "alg": "RS256",
- ["kid": "xxxxx"](https://tools.ietf.org/html/rfc7515#section-4.1.4)
+ "kid": "<key-identifier>"
 }
 ```
 
 Payload:
 ```
 {
- "iss": "https://\<issuer website\>/",
- "sub": "<someone@someone.com>",
- "idp": "google",
+ "iss": "https://<issuer-website>/",
+ "sub": "<subject-identifier>",
+ "idp": "<short-idp-identifier>",
  "aud": [
-  "client_id",
-  "client_id2"
+  "<client-id1>",
+  "<client-id2>" ...
  ],
- "iat": 1553545136,
- "exp": 1553631536,
- "jti": "xxxx-xxxx-xxxx",
- "scope": "openid \<ga4gh-spec-scopes\>",
- "ga4gh_userinfo_claims": ["claim_name_1", "claim_name_2.substructure_name"],
+ "iat": <seconds-since-epoch>,
+ "exp": <seconds-since-epoch>,
+ "jti": <token-identifier>,
+ "scope": "openid <ga4gh-spec-scopes>"
 }
 ```
--   iss: MUST be able to be appended with .well-known/openid-configuration to
-    get spec of broker.
+-   `iss`: REQUIRED. MUST be able to be appended with
+    .well-known/openid-configuration to get spec of Broker.
 
--   sub: authenticated user unique identifier. A broker MAY abstract the suggested user email address with a unique identifier provided it maintains a way to map the user. See the ELIXIR for an implementation example. The sub in this case is a unique identifier issued by ELIXIR that abstracts the user's "real" email. Downstream Clearinghouses will need to know how to handle the `sub` attribute.
+-   `sub`: REQUIRED. Authenticated user unique identifier.
 
--   idp: (optional) SHOULD contain the IDP the user used to auth with. Such as
-    "Google". This does not have to be unique and can be used just to help
-    inform if that’s what a data owner or data holder needs.
+-   `idp`: OPTIONAL. SHOULD contain the IDP the user used to auth with.
+    A non-normative example is "google". This does not have to be unique and
+    can be used just to help inform if that is what a data owner or data holder
+    needs.
 
--   aud: MUST contain the Oauth Client ID of the relying party except when
-    allowing the token to be used as an open federation Embedded Token where
-    `aud` MUST NOT be included. When included, it MAY contain other strings or
-    identifiers as well.
+-   `aud`: REQUIRED. MUST contain the Oauth Client ID of the relying party.
 
--   iat: time issued
+-   `iat`: REQUIRED. Time issued.
 
--   exp: time expired
+-   `exp`: REQUIRED. Time expired.
 
--   jti: RECOMMENDED. a unique identifier for the token as per
+-   `jti`: RECOMMENDED. a unique identifier for the token as per
     [RFC7519 Section 4.1.7](https://tools.ietf.org/html/rfc7519#section-4.1.7)
 
--   scope: scopes verified. Must include "openid". Will also include any
-    \<ga4gh-spec-scopes\> needed for the GA4GH compliant environment (e.g.
-    "ga4gh" is the [scope for RI
+-   `scope`: REQUIRED. Includes verified scopes. MUST include "openid". Will also
+    include any `<ga4gh-spec-scopes>` needed for the GA4GH compliant environment
+    (e.g. "ga4gh" is the [scope for RI
```

Comment:
The RI CLaims spec uses "ga4gh_passport_v1" both for the scope and for the claim names.

Reply:
Done